httplug/application-developers.rst

```diff
@@ -1,11 +0,0 @@
-HTTPlug for application developers
```

Comment:
this was only linking to the httplug bundle so it becomes redundant with the integrations chapter.